peps/pep-0661.rst

```diff
@@ -421,6 +394,25 @@ idiom were unpopular, with the highest-voted option being voted for by only
 25% of the voters.
 
 
+Allowing customization of repr
+------------------------------
+
+This was desirable to allow using this for existing sentinel values without
+changing their repr.  However, this was eventually dropped as it wasn't
+considered worth the added complexity.
+
+
+Using ``typing.Literal`` in type annotations
+--------------------------------------------
+
+This was suggested by several people in discussions and is what this PEP
+first went with.  However, it was pointed out that this would cause potential
+confusion, due to e.g. ``Literal["MISSING"]`` referring to the string value
+``"MISSING"`` rather than being a forward-reference to a sentinel value
+``MISSING``.  Using the bare name was also suggested often in discussions, and
```

Comment:
It's also just simpler and doesn't require an import.

Reply:
Good points :+1: